src/hooks/useWalletENS.ts

```diff
@@ -0,0 +1,58 @@
+import { useCallback } from 'react';
```

Comment:
This hook seems only specific for avatars atm, maybe should be named `useWalletENSAvatar`?

Reply:
👌  yeah